### ISSUE TITLE
[Docs] TypeError when creating Dynamics from data

### ISSUE THREAD

**Issue by shik-design** [lisphilar/covid19-sir#1113]
## What we need to document?
I have successfully downloaded the data using 
 
```
downloader = cs.DataDownloader(directory="input", update_interval=24, verbose=1)
# All country-level data
downloader.layer()
df = downloader.layer(country="Japan", province=None)
```

but:  `dynamics = cs.Dynamics(model=cs.SIRF, data=df)`
gives the following error message 

```

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
~\AppData\Local\Temp/ipykernel_11728/1803869195.py in <module>
      1 # Select country name and register the data
----> 2 dynamics = cs.Dynamics(model=cs.SIRF, data=df)

TypeError: __init__() got an unexpected keyword argument 'data
```

**Comment by lisphilar** [comment]
Please use class method `cs.Dynamics.from_data()`.

**Comment by lisphilar** [comment]
Please confirm the data includes Susceptible and Infected additionally. `DataEnginneer` can create these cuolumns, but the output of `DataDownloader().layer()` does not have these columns as explained in the notebooks.

**Comment by shik-design** [comment]
# FORMER USAGE
```
import covsirphy as cs
# Download and update datasets
loader = cs.DataLoader("input")
data_dict = loader.collect()
# Select country name and register the data
snl = cs.Scenario(country="Japan")
snl.register(**data_dict)
# Check records
snl.records()
# S-R trend analysis
snl.trend().summary()
# Parameter estimation of SIR-F model
snl.estimate(cs.SIRF)
# History of reproduction number
snl.history(target="Rt")
# History of parameters
snl.history_rate()
snl.history(target="rho")
# Baseline
snl.add(days=30)
# Forecasting
snl.predict(days=30)
snl.adjust_end()
snl.rename(old="Multivariate_regression_Likely", new="Likely")
snl.delete_matched(pattern=r"^Multi")
# Compare scenarios
snl.describe()
# Summary
snl.summary()
# Simulation
snl.simulate()
```
# CURRENT USAGE (Version 2.40.0)
Please for the sake of clarity, kindly outline the commands required as above. It may be useful for other researchers also.

**Comment by lisphilar** [comment]
OK, I will update [GitHub Page](https://lisphilar.github.io/covid19-sir/) immediately after the next relase of stable version. The new classes are implemented in the latest development version and I'm writing notebook in [example directory](https://github.com/lisphilar/covid19-sir/tree/master/example).

**Comment by shik-design** [comment]
OK. It's noted with gladness

**Comment by lisphilar** [comment]
Released new stable version 2.26.0! I also updated README and documentation.
